packages/core/src/order/order-request-body.ts

```diff
@@ -52,7 +53,7 @@ export type OrderPaymentInstrument =
     | (CreditCardInstrument & WithDocumentInstrument)
     | (CreditCardInstrument & WithCheckoutcomFawryInstrument)
     | (CreditCardInstrument & WithCheckoutcomSEPAInstrument)
-    | (CreditCardInstrument & WithCheckoutcomiDealInstrument)
+    | (CreditCardInstrument & IdealInstrument)
```

Comment:
```suggestion
    | (CreditCardInstrument & WithIdealInstrument)
```
Could we still use the `With` naming pattern?
The rest looks good to me.

Reply:
@bc-peng I decided to remove "With" part, because we started to use IdealInstrument interface separately from CreditCardInstrument. Do we still need "with" in this case?

Reply:
We can create WithIdealInstrument that would be a identical to the IdealInstrument. But it seems like overhead.